src/libecalc/core/graph_result.py

```diff
@@ -631,6 +649,24 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
                                     rate_type=RateType.STREAM_DAY,
                                     regularity=regularity.for_timesteps(model.timesteps).values,
                                 ),
+                                standard_rate_sm3_per_day=TimeSeriesRate(
+                                    timesteps=model.timesteps,
+                                    values=stage_result.inlet_stream_condition.standard_rate_sm3_per_day
+                                    if stage_result.inlet_stream_condition.standard_rate_sm3_per_day is not None
+                                    else [math.nan] * len(model.timesteps),
+                                    unit=Unit.STANDARD_CUBIC_METER_PER_DAY,
+                                    rate_type=RateType.STREAM_DAY,
+                                    regularity=regularity.for_timesteps(model.timesteps).values,
+                                ),
+                                standard_rate_before_asv_sm3_per_day=TimeSeriesRate(
+                                    timesteps=model.timesteps,
+                                    values=stage_result.inlet_stream_condition.standard_rate_before_asv_sm3_per_day
```

Comment:
```suggestion
                                    values=stage_result.outlet_stream_condition.standard_rate_before_asv_sm3_per_day
```

Reply:
The standard_rate_before_asv has been corrected to the outlet_stream_conditions.